source/server/LambdaLocationUpdateHandler.js

```diff
@@ -73,7 +78,12 @@ exports.handler = (event, context, callback) => {
 // This function simply reports an error back to the client.
 function abortLocationUpdate(reason, callback) {
     callback(null, {
-        "status" : "error",
-        "message" : reason
+        // `statusCode` : 201 indicates error.
+        "statusCode" : 201,
```

Comment:
I am SO SORRY for being picky, but technically even if an error occurs, the HTTP response code should still be `200` because it shows a good response to the HTTP request.

Even though on an application-wide level the request failed because of some bad latitude and longitude parameters to the server, on a network layer, the server did it's job successfully so should return `200`.

The `200` refers to an HTTP response code, not an error code for our application. `201` actually means something really different:

>HTTP Status Code 201: The request has been fulfilled and has resulted in one or more new resources being created.

So if you just change the statusCode to `200` then this all looks great!